src/desktop/apps/artwork/components/commercial/view.coffee

```diff
@@ -39,14 +40,18 @@ module.exports = class ArtworkCommercialView extends Backbone.View
   acquire: (e) ->
     e.preventDefault()
 
-    order = new PendingOrder
-    @form = new Form $form: @$('form'), model: order
+    loggedInUser = CurrentUser.orNull()
+    if loggedInUser && loggedInUser.hasLabFeature('New Buy Now Flow')
```

Comment:
this is coffeescript (ugh), so can just be `if CurrentUser.orNull()?.hasLabFeature(...`